infra/modules/azure_api_management/tests/apim.tftest.hcl

```
@@ -76,6 +67,11 @@ run "apim_is_correct_plan" {
     condition     = length(azurerm_api_management_logger.this) > 0
     error_message = "The APIM logger does not exist"
   }
+
+  assert {
+    condition     = length(azurerm_api_management.this.zones) == 2 && contains(azurerm_api_management.this.zones, "1") && contains(azurerm_api_management.this.zones, "2")
+    error_message = "The APIM zones are incorrect, they should be ['1', '2']"
```

question: I cannot understand this condition which looks like:

```suggestion
    condition     = azurerm_api_management.this.zones == ["1", "2"]
    error_message = "The APIM zones are incorrect, they should be ['1', '2']"
```

can't zones just be hardcoded in case?

---

This causes the test to fail even if the configuration is correct because the state does not list the zones exactly as in the expected list, making verification unreliable